### PR TITLE
Document saturation arithmetic and min/max fields

### PR DESCRIPTION
**Pull request by turbolent**
- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/language/values-and-types.mdx [hunk 1]
```diff
@@ -232,6 +232,49 @@ Fixed-Point numbers have multiple built-in functions you can use.
   fix.toBigEndianBytes()  // is `[0, 0, 0, 0, 7, 84, 212, 192]`
   ```
 
+## Minimum and maximum values
+
+The minimum and maximum values for integer and fixed-point number types are available through the fields `min` and `max`.
```
Comment (joshuahannan) [inline docs/language/values-and-types.mdx:237]: Is it worth mentioning they are available for **all** integer and fixed point numbers? Or giving more examples?

**Review by joshuahannan — APPROVED** [review]
LGTM

File: docs/language/values-and-types.mdx (hunk 1 above)
Comment (turbolent) [inline docs/language/values-and-types.mdx:237]:
```suggestion
The minimum and maximum values for all integer and fixed-point number types are available through the fields `min` and `max`.
```

Comment (turbolent) [reply docs/language/values-and-types.mdx]: Clarified in 5de30cdee2e4918197fadab2e2356a7f66e71713

**Comment by turbolent** [comment]
@joshuahannan added another example in abf6f9e2b48ce06ca514ea87dfdadc4243d7beaa

**Review by SupunS — APPROVED** [review]
Nice!

**Pull request merged**